public/styles/styles.less

```diff
@@ -784,7 +785,7 @@ div#board-wrap {
   }
 
   .bets {
-    font-size: 1.2em;
+    font-size: 2em;
```

Comment:
Nice, we can actually also turn `pot` size into white. 
For me the text size (bets and pots) also little bit bigger